posts/2021-02-19-mpcontextpropagation-requesttiming-21003-beta.adoc

```diff
@@ -0,0 +1,323 @@
+author_picture: https://avatars3.githubusercontent.com/jakub-pomykala
+author_github: https://github.com/jakub-pomykala
+seo-title: MicroProfile Context Propagation 1.2, RequestTiming 1.0 and JakartaMail 2.0 in Open Liberty 21.0.0.3-beta - OpenLiberty.io
+seo-description: Open Liberty 21.0.0.3-beta contains four All Beta features (new `expandLocation` property for App Manager configuration, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature (RequestTiming 1.0), MicroProfile Context Propagation 1.2 and Automatic Cleanup of Leaked Connections) and one feature in Jakarta EE 9 (JakartaMail).
+blog_description: Open Liberty 21.0.0.3-beta contains four All Beta features (new `expandLocation` property for App Manager configuration, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature (RequestTiming 1.0), MicroProfile Context Propagation 1.2 and Automatic Cleanup of Leaked Connections) and one feature in Jakarta EE 9 (JakartaMail). Also in 21.0.0.3-beta the link:https://github.com/OpenLiberty/open-liberty/issues/15649[issue in MicroProfile Fault Tolerance 3.0] from 21.0.0.2-beta has now been resolved meaning that the `mpFaultTolerance-3.0` feature and the `microProfile-4.0` convenience feature can be used in the Open Liberty 21.0.0.3-beta.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= MicroProfile Context Propagation 1.2, RequestTiming 1.0 and JakartaMail 2.0 in Open Liberty 21.0.0.3-beta
+Jakub Pomykala <https://github.com/jakub-pomykala>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+'''
+
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<allbeta, All Beta Features>>: a larger package that contains all Open Liberty beta features (including Jakarta EE 9 beta features) and GA features and functions.
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools.
+
+If you try either package, <<feedback, let us know what you think>>.
+
+In addition to all Jakarta EE 9 and MicroProfile 4.0 beta features, Open Liberty 21.0.0.3-beta also contains five brand new features.
+
+'''
+
+[#allbeta]
+== All Beta Features package
+
+The All Beta Features package includes the following beta features:
+
+* <<MPContext, MicroProfile Context Propagation 1.2>>
+* <<requestTiming, Ability to control whether thread dumps are collected when a hung request is detected in the Request Timing feature>>
+* <<leakedConnections, Automatic Cleanup of Leaked Connections>>
+* <<expandLocation, expandLocation property for App Manager configuration>>
+
+
+These features also join MicroProfile 4.0 features which can be found link:https://openliberty.io/blog/2021/01/26/ee9-messaging-security-21002-beta.html#allbeta[here].
```

Comment:
This is not quite straightforward as the previous beta blog removed MicroProfile Fault Tolerance 3.0. The 21.0.0.3-beta has the full MP 4.0 features. I suggest listing all of lists to make it complete. Besides, we would like to use this beta blog for GA blog.

Reply:
Done